src/en/config-azure.md

```diff
@@ -1,123 +1,178 @@
+
+![azure_settings_page](media/config-azure-stable_settings-page.png)
+
+### `storage-account-name`
+
+First create a *storage account*. In the left pane, scroll down to 'Storage'
+and press the **+ NEW** icon (bottom-left). An overlay page will appear. Click
+'Quick Create' and fill in the 'URL' field (e.g. `jujuazure`). This is the
+value to be used for `storage-account-name`.
+
+![azure_storage account_creation](media/config-azure-stable_storage.png)
+
+### `location`
 
-The generic configuration sections generated for Windows Azure will look
-something like this:
+In the same dialog, select a 'Location/Affinity Group'.
+
+Note that there is a limited set of regions available in the Azure UI. Choose
+one that is closest to you:
+
+![azure_storage_region_dropdown](media/config-azure-stable_storage_locations_dropdown.png)
+
+For insight into the 'Replication' field see
+[Azure storage redundancy documentation](https://azure.microsoft.com/documentation/articles/storage-redundancy)
+.
+
+!!! Note: Once you bootstrap Juju, an Azure affinity group (e.g.
+`juju-azure-ag`) will appear in this list if you ever come back to it. See
+[stackoverflow.com: "Azure Availability Set vs Affinity
+Group"](http://stackoverflow.com/questions/25472549/azure-availability-set-vs-affinity-group)
+for context.
+
+### `management-certificate-path`
+
+The value of this parameter is the file path of the private key (and
+certificate) stored in the PEM file created earlier:
+`/home/ubuntu/.juju/juju-azure.pem`.
+
+
+## Confirm configuration and bootstrap
+
+According to all the above, the Azure section of file `environments.yaml` for
+this example would look like this (comments removed for simplicity):
 
 ```yaml
-# https://jujucharms.com/docs/config-azure.html
     azure:
         type: azure
-        # location specifies the place where instances will be started,
-        # for example: West US, North Europe.
-        #
-        location: West US
-        # The following attributes specify Windows Azure Management
-        # information. See:
-        # http://msdn.microsoft.com/en-us/library/windowsazure
-        # for details.
-        #
-        management-subscription-id: 00000000-0000-0000-0000-000000000000
-        management-certificate-path: /home/me/azure.pem
-        # storage-account-name holds Windows Azure Storage info.
-        #
-        storage-account-name: abcdefghijkl
-        # force-image-name overrides the OS image selection to use a fixed
-        # image for all deployments. Most useful for developers.
-        #
-        # force-image-name: b39f27a8b8c64d52b05eac6a62ebad85__Ubuntu-13_10-amd64-server-DEVELOPMENT-20130713-Juju_ALPHA-en-us-30GB
-        # image-stream chooses a simplestreams stream to select OS images
-        # from, for example daily or released images (or any other stream
-        # available on simplestreams).
-        #
-        # image-stream: "released"
+        location: Central US
+        storage-account-name: jujuazure
```

Comment:
Confusingly, Juju 1.24 bootstrap requires 'storage-account-name'.

ERROR juju.cmd supercommand.go:429 there was an issue examining the environment: storage-account-name: expected string, got nothing